mycroft/skills/intent_service.py

```diff
@@ -254,6 +258,18 @@ def send_metrics(self, intent, context, stopwatch):
         report_timing(ident, 'intent_service', stopwatch,
                       {'intent_type': intent_type})
 
+    def _get_transformers_context(self, message, lang):
```

Comment:
Consider refactoring to `_transform_utterance`, `_transform_add_context` or something that indicates this might transform the utterance in addition to providing more context